exchanges/exchange.go

```diff
@@ -966,7 +966,7 @@ type GetOrdersRequest struct {
 	StartTicks time.Time
```

Comment:
OrderType and OrderSide can be redefined using ioate enum and generate Stringer interface using go:generate

Reply:
Out of scope for this PR. You can open a PR with those changes if you want.